test/libsolidity/syntaxTests/inheritance/interface/multiple_parents.sol

```diff
@@ -0,0 +1,15 @@
+interface SuperA {
+    function test() external returns (uint256);
+    function testA() external returns (int128);
+}
+
+interface SuperB {
+    function test() external returns (uint256);
+    function testB() external returns (int256);
+}
+
+interface Sub is SuperA, SuperB {
+}
+
+// ----
+// TypeError: (236-271): Derived contract must override function "test". Two or more base classes define function with same name and parameter types.
```

Comment:
This error is kind of correct, but misleading, since once you do override those functions you'll get the error `interfaces can't override`

Reply:
Yeah, I figured this would have to be changed.

Reply:
Can this now be fixed without errors or not?

Reply:
Done.